src/dao/index.ts

```diff
@@ -0,0 +1,22 @@
+import { MongoClient, Db } from 'mongodb';
+require('dotenv').config();
+export * from './types';
+
+export let client: MongoClient;
+export let database: Db;
+
+export const connect = async (): Promise<Db> => {
+  if (!database) {
+    try {
+      console.info(`Connecting to database ${process.env.MONGO_DB_URI}`);
+      client = await MongoClient.connect(process.env.MONGO_DB_URI!, {
+        useUnifiedTopology: true
+      });
+      database = client.db('kotkl');
+    } catch (error) {
+      console.log('Unable to connect to DB', error);
+    }
+  }
+  console.log('Connected');
```

Comment:
Maybe change this to console.info to match the "Connecting to database ... message".

Reply:
changed all console.logs to console.info and updated successful connection string